activemq-unit-tests/src/test/java/org/apache/activemq/command/ActiveMQMessageTest.java

```diff
@@ -193,7 +193,7 @@ public void testGetAndSetJMSMessageID() throws Exception {
         assertEquals(msg.getJMSMessageID(), this.jmsMessageID);
     }
 
-    public void testGetAndSetJMSTimestamp() {
+    public void testGetAndSetJMSTimestamp() throws JMSException {
```

Comment:
I believe you can remove `throws JMSException` from these tests again.

Reply:
Done